carbon-design-system/carbon-addons-iot-react
# 3295 card padding prop

Closes #3295 

**Summary**

- adds prop to remove padding from card content

**Change List (commits, features, bugs, etc)**

- `padding` prop for cards with `default` and `none` options
- added to mdx
- added knob basic and custom card stories
- added tests to confirm class

**Acceptance Test (how to verify the PR)**

- check the [implementing a custom card story](https://deploy-preview-3296--carbon-addons-iot-react.netlify.app/?path=/story/1-watson-iot-card--implementing-a-custom-card)
- set padding knob to 'default'
- confirm padding is re-added around the table
- setting padding knob to 'none'
- confirm padding is removed.

**Regression Test (how to make sure this PR doesn't break old functionality)**

- make sure default gets applied automatically to all other cards and doesn't disrupt current visuals



**Things to look for during review**
